CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ Thank you for your interest in contributing to the Uuid Project!
 * [Helpful Links](#helpful-links)
 
 For any questions, please make a post on [users.rust-lang.org][u-r-l-o], post
-on [uuid-rs mailing list] or join our [gitter] channel.
+on [uuid-rs mailing list].
```

Comment:
I think we aren't using the mailing list anymore.

Reply:
I think instead we can unit GitHub Discussions at some point in the future.

Reply:
I've used Discussions elsewhere and they've been good for keeping issues focused on actionable things. I'd be keen to use them here in `uuid` too